context/schema.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
This is a copy of the schema now included in the ProvVis directory? Do we need this here? If so, could we make the file name more descriptive if it's not much trouble? Do we have other components which require this schema?

Reply:
Not sure why this ended up here. Deleting, and giving a fuller name for the other copy.